### DISCUSSION TITLE
How to get query string params in `auth` when my application is first accessed?

### DISCUSSION THREAD

**Discussion by khteh** [nextauthjs/next-auth#7080] [Help]
### Question 💬

https://github.com/nextauthjs/next-auth/issues/7077

How do I get the initial query string params in https://next-auth.js.org/configuration/initialization#advanced-initialization when the user first accesses my web application? I want to customize cookie only when my web application is accessed with `?sid=<GUID>` query string param: `https://www.myapp.com?sid=GUID`

### How to reproduce ☕️

https://github.com/nextauthjs/next-auth-example runs inside an iFrame

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by raphaelnoguier** [comment]
Hey @khteh did you found any workaround with this  ? I'm facing the same issue?

Thanks :)

**Comment by eibrahimov** [comment]
Hello @khteh, I am experiencing the same issue as you. Have you found any solution yet?
